python/fbprophet/forecaster.py

```diff
@@ -34,10 +29,18 @@
     plot_seasonality,
 )
 
+import fbprophet.hdays as hdays_part1
+import holidays as hdays_part2
+
 logging.basicConfig()
 logger = logging.getLogger(__name__)
 warnings.filterwarnings("default", category=DeprecationWarning)
 
+try:
+    import pystan  # noqa F401
+except ImportError:
+    logger.exception('You cannot run fbprophet without pystan installed')
```

Comment:
There is no point checking pystan here because if it were not available, `from fbprophet.models` above would fail. This message was meant to anticipate the failure to unpickle the pystan model in `models.py`.

Reply:
We needed this for some internal tests which are no longer being used so at this point I think we can just remove the check and raise the ImportError. We could also wrap the unpickling in a try/except if we wanted a more informational message about PyStan not working on that error.

Reply:
I'm OK with removing this.